python/ray/ray_constants.py

```diff
@@ -154,6 +154,12 @@ def to_memory_units(memory_bytes, round_up):
 
 RESOURCES_ENVIRONMENT_VARIABLE = "RAY_OVERRIDE_RESOURCES"
 
+# Default number of cpu instruction set resources.
+NUM_OF_CPU_RESOURCE = 512.0
+
+# Default list of cpu instruction set, like "avx, avx2".
+CPU_INSTRUCTION_SET = ""
```

Comment:
```suggestion
CPU_INSTRUCTION_SETS = ""
```
Since here you may have many instruction sets.

Reply:
I'm considering what the best name here is. Actually, this is used as a hints that tell node that if we have these sets, enable them as resources to be scheduled.